02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,47 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>
+    Justin's Bio
+  </title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="css/style.css"/>
 </head>
 <body>
+  <h1>
+    - Justins Bio Page -
+  </h1>
+  <hr>
+
+  <img src="https://scontent-dft4-2.xx.fbcdn.net/t31.0-8/1271524_3504005055325_1637188001_o.jpg" alt="Justin" style="width: 80%; height: 80%;">
+
+  <!-- Main Section -->
+  <div id="main">
+    <header>Bio</header>
+      <p>This is a lengthy bio, telling you stuff about things. This is a lengthy bio, telling you stuff about things. This is a lengthy bio, telling you stuff about things. This is a lengthy bio, telling you stuff about things. This is a lengthy bio, telling you stuff about things. This is a lengthy bio, telling you stuff about things. This is a lengthy bio, telling you stuff about things. </p>
+    <header>Movies</header>
+      <p>Inception, Lucky Number Slevin, Sicario, Fury, Inglorious Basterds, Gran Torino, No Country For Old Men, Birdman, Fargo</p>
```

Comment:
Indentation: After closing `header` tag, the following opening tag needs to be on the same level. 
I would recommend to always put block element tags on their own line: 
```html
<header>
   Movies
</header>
```
That way you see immediately where to indent and by how much